streams/src/main/java/org/apache/kafka/streams/state/QueryableStoreType.java

```diff
@@ -44,7 +44,7 @@
      * @param storeProvider     provides access to all the underlying StateStore instances
      * @param storeName         The name of the Store
      * @return a read-only interface over a {@code StateStore}
-     *        (cf. {@link org.apache.kafka.streams.state.QueryableStoreTypes.KeyValueStoreType})
+     *        (cf. {@link QueryableStoreTypes.KeyValueStoreType})
```

Comment:
This class is in the same package, so the fully-qualified name is not necessary.